src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -2798,12 +2804,17 @@ protected override void RemoveItem(string path, bool recurse)
                     return;
                 }
 
+#if !UNIX
```

Comment:
I can move some code around so the UNIX code is in one place and the !UNIX code is also together.